src/citrine/resources/design_space.py

```diff
@@ -206,6 +227,54 @@ def create_default(self,
         data = self.session.post_resource(path, json=payload, version=self._api_version)
         return self.build(DesignSpace.wrap_instance(data["instance"]))
```

Comment:
Have you confirmed that this works as expected with a hierarchical design space?

Reply:
Yep! I've been running all of this code to test the serde, and it does indeed work.